classes/custompage.php

```diff
@@ -98,10 +94,22 @@ public function update($data) {
      * @param string $item
      * @return mixed
      */
-    public function __get($item) {
+    public function __get($item): mixed {
         if (isset($this->_data->$item)) {
             return $this->_data->$item;
         }
+        return null;
+    }
+
+    public function __set($item, $value): void {
+        $this->_data->$item = $value;
+    }
+
+    public function __isset($item): bool {
+        return isset($this->_data->$item);
+    }
```

Comment:
This one as well
Example of the parent methods https://www.php.net/manual/en/language.oop5.overloading.php#object.set

```suggestion
    public function __isset(string $name): bool {
        return isset($this->_data->$name);
    }
```